### PR TITLE
Fix #174: Add signature prompt type and fix survey prompt type URL in docs

### PR DESCRIPTION
**Pull request by EstherPhebe**
Add signature prompt type and fix survey prompt type URL in docs docs

closes odk-x/tool-suite-X#174

#### What is included in this PR?
1. URL [https://docs.odk-x.org/xlsx-converter-reference/#id4](https://docs.odk-x.org/xlsx-converter-reference/#id4) improved to [https://docs.odk-x.org/xlsx-converter-reference/#survey-prompt-types](https://docs.odk-x.org/xlsx-converter-reference/#survey-prompt-types) 
2. Signature prompt type and description added to Survey Prompt Types list.

#### What problems did you encounter?
I built and served the cloned repository with Docker, I noticed that when I ran `.\run-task.bat odkx-autobuild`  it returned  `docker: invalid reference format`. I was tinkering around in the `run-task.bat` file and found out that when I added double quotes around `'%~dp0:/mnt'` , the problem was solved. I spoke to the mentors and @wbrunette asked that I include this in the Pull Request with the Windows Version.

### CODE REVIEW

File: run-task.bat [hunk 1]
```diff
@@ -1 +1 @@
-docker run --rm -v %~dp0:/mnt -p 8080:8080 --name odkx-docs odkx-docs %1
+docker run --rm -v "%~dp0:/mnt" -p 8080:8080 --name odkx-docs odkx-docs %1
```
Comment (wbrunette) [inline run-task.bat:1]: Can you submit your docker change in a separate PR? It's best to separate issues, the top part solve issue #174 the bottom part is different. You can create different pull requests by having different branches.

**Review by wbrunette — CHANGES_REQUESTED** [review]

File: run-task.bat (hunk 1 above)
Comment (EstherPhebe) [reply run-task.bat]: Seen, will sort it out. Thanks.

**Review by wbrunette — APPROVED** [review]

File: odkx-src/xlsx-converter-reference.rst [hunk 2]
```diff
@@ -374,6 +375,9 @@ The following prompt types are available in ODK-X Survey.
       | to the user, and allows the user to click one item. One of the
       | choices provided is an other option which if clicked provides a text
       | box for the user to enter a value.
+  * - signature
+    - | Used to capture the user signature and allows them to enter a signature
```
Comment (linl33) [inline odkx-src/xlsx-converter-reference.rst:379]: > user signature

It is not necessarily the user's signature, it could be anyone's signature. I think the docs shouldn't specify whose signature it captures to avoid confusion.

Comment (elmps2018) [reply odkx-src/xlsx-converter-reference.rst:379]: Agree with @linl33. I think "Used to capture a signature" or "Used to enter a signature" is simpler/clearer.

File: odkx-src/xlsx-converter-reference.rst [hunk 3]
```diff
@@ -374,6 +375,9 @@ The following prompt types are available in ODK-X Survey.
...
+      | by tracing with the finger.
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-reference.rst:380]: Maybe "by touching/tracing on the tablet screen" would be clearer here.

**Review by elmps2018 — COMMENTED** [review]
Great to see this getting documented!

File: odkx-src/xlsx-converter-reference.rst (hunk 2 above)
Comment (EstherPhebe) [reply odkx-src/xlsx-converter-reference.rst:379]: Okay, will make the changes.
